src/instructlab/eval/mmlu.py

```diff
@@ -1,24 +1,30 @@
 # SPDX-License-Identifier: Apache-2.0
 
 # Local
-from .evaluator import Evaluator
+from instructlab.eval.evaluator import Evaluator
 
+# Third Party 
+from lm_eval.evaluator import simple_evaluate
+import os
 
 class MMLU_Evaluator(Evaluator):
     """
     Child class of an Evaluator for Massive Multitask Language Understanding (MMLU)
 
     Attributes:
+        model_path   absolute path to or name of a huggingface model
         tasks        list of tasks for MMLU to test the model with
+        model_dtype  dtype of model when served
         few_shots    number of examples
         batch_size   number of GPUs
     """
 
     def __init__(
-        self, model_path, tasks: list[str], few_shots: int = 2, batch_size: int = 5
+        self, model_path, tasks: list[str], model_dtype = 'bfloat16', few_shots: int = 2, batch_size: int = 5
     ) -> None:
         super().__init__(model_path)
         self.tasks = tasks
```

Comment:
There should maybe be a default for tasks, overridable by the constructor param. 

In an ideal world, a user might just call:

```python=
MODEL_PATH = args.model_name_or_path # or a model object, might be more convenient for our library

# with defaults for tasks, model type from model object, sensible defaults otherwise
mmlu = MMLU_Evaluator(model_path) 
overall_score, individual_scores = mmlu.run()
```

Reply:
Elsewhere, there could be a `MMLUTasks` object something like:

```
from pydantic import BaseModel, Field

class MMLUTasks(BaseModel):
    mmlu_abstract_algebra: bool = Field(default=True)
    mmlu_anatomy: bool = Field(default=True)
    mmlu_astronomy: bool = Field(default=True)
    ....
```

Then the user could specify which tasks to include vs. not.

Reply:
oh the evaluation harness handles grabbing the model, nvm I see about the "model_name_or_path thing.

Reply:
@JamesKunstle we'd discussed this quite a bit when we first started working on the library and decided we want the caller to the library to pass in the tasks list and not have a default tasks list or something preset.

Reply:
@cdoern is going to handle parameterizing the defaults in the cli. Also, a user won't need to worry about changing the defaults - it should always be a standardized set for MMLU, but I could see that case being made for MMLUBench